### PR TITLE
CID:1428 Route Dora through events broker

### PR DESCRIPTION
**Pull request by alfredo-mfaria**

### CODE REVIEW

File: src/main/kotlin/net/leanix/vsm/githubbroker/connector/adapter/feign/FeignAssignmentProvider.kt [hunk 1]
```diff
@@ -8,11 +8,10 @@ import org.springframework.stereotype.Component
 class FeignAssignmentProvider(private val vsmClient: VsmClient) : AssignmentProvider {
     override fun getAssignments(
         integrationName: String,
-        configSetName: String,
-        brokerVersion: String
+        configSetName: String
     ): Result<List<Assignment>> {
         return kotlin.runCatching {
-            vsmClient.getAssignments(integrationName, configSetName, brokerVersion).toDomain()
+            vsmClient.getAssignments(integrationName, configSetName).toDomain()
```
Comment (geoandri) [inline src/main/kotlin/net/leanix/vsm/githubbroker/connector/adapter/feign/FeignAssignmentProvider.kt:14]: Just out of curiosity, no need to get the `brokerVersion` anymore?

**Review by geoandri — APPROVED** [review]

**Comment by geoandri** [comment]
LGTM. 

So the part of creating the CHANGE events has been delegated to the Events broker? Is this correct?

File: src/main/kotlin/net/leanix/vsm/githubbroker/connector/adapter/feign/FeignAssignmentProvider.kt (hunk 1 above)
Comment (alfredo-mfaria) [reply src/main/kotlin/net/leanix/vsm/githubbroker/connector/adapter/feign/FeignAssignmentProvider.kt:14]: we do, but I implement it in the interceptor for all requests:
https://github.com/leanix/vsm-github-broker/blob/ce7209cab40dcbd1de9b57522de3fe03bef9adc7/src/main/kotlin/net/leanix/vsm/githubbroker/shared/auth/adapter/feign/config/MtmFeignClientConfiguration.kt#L15

**Comment by alfredo-mfaria** [comment]
> LGTM.
> 
> So the part of creating the CHANGE events has been delegated to the Events broker? Is this correct?

You are correct. This was a suggestion from Henrique to reduce the amount of calls :)

**Pull request merged**